build/test.mk

```diff
@@ -49,7 +49,7 @@ test: test-get-envtools ## Runs unit tests, excluding kubernetes controller test
 .PHONY: test-get-envtools
 test-get-envtools:
 	@echo "$(ARROW) Installing Kubebuilder test tools..."
-	$(call go-install-tool,$(ENV_SETUP),sigs.k8s.io/controller-runtime/tools/setup-envtest@latest)
+	$(call go-install-tool,$(ENV_SETUP),sigs.k8s.io/controller-runtime/tools/setup-envtest@v0.0.0-20240521074430-fbb7d370bebc)
```

Comment:
Can you create an issue for the following (if you didn't already):

We should either:

- Revert this in the future and use `latest` when they fix the issue.
- Move this into a place where depdendabot can manage it for it. 

We don't want this to be hardcoded forever.

Reply:
Sure, just opened a bug here: https://github.com/radius-project/radius/issues/7630. I also opened one in the controller-runtime repo that I'll monitor for responses: https://github.com/kubernetes-sigs/controller-runtime/issues/2838

Reply:
Looks like #7630 is tracking the build-break. We should also have a plan to remove the workaround.

Reply:
I considered them parts of the same problem but I updated the issue to explicitly call out removing the workaround